src/sink_stream.c

```diff
@@ -57,12 +58,16 @@ static int stream_formatter(FILE *pipe, void *data, metric_type type, char *name
             STREAM("%s%s.upper|%f|%lld\n", prefix, name, timer_max(&t->tm));
             STREAM("%s%s.count|%" PRIu64 "|%lld\n", prefix, name, timer_count(&t->tm));
             for (i=0; i < ct->global_config->num_quantiles; i++) {
-                if (ct->global_config->quantiles[i] == 0.5) {
+                int percentile;
+                double quantile = ct->global_config->quantiles[i];
+                if (quantile == 0.5) {
                     STREAM("%s%s.median|%f|%lld\n", prefix, name, timer_query(&t->tm, 0.5));
                 }
-                STREAM("%s%s.p%0.0f|%f|%lld\n", prefix, name,
-                    ct->global_config->quantiles[i] * 100,
-                    timer_query(&t->tm, ct->global_config->quantiles[i]));
+                if (to_percentile(quantile, &percentile)) {
+                    syslog(LOG_ERR, "Invalid quantile: %lf", quantile);
```

Comment:
config parser already performs a sanity check on the incoming quantiles list. going to lose this check